src/components/Modals/SelectModals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -40,6 +40,23 @@ export const SelectTokenModal: React.FC<ISelectTokenModal> = ({
     USDC: 'USD Coin'
   }
 
+  const selectTokens = (name: string): number => {
```

Comment:
nope, on different networks we have different sets of tokens, try to implement something more universal